packages/tradershub/src/flows/FirstSignup/CitizenshipModal/CitizenshipModal.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import { useFormikContext } from 'formik';
+import { Modal } from '@/components';
+import { useClientCountry, useResidenceList } from '@deriv/api';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Button, Checkbox, Dropdown, Text } from '@deriv-com/ui';
+import { TFirstSignupFormValues } from '../FirstSignupWrapper/FirstSignupWrapper';
+
+type TCitizenshipModal = {
+    onClickNext: VoidFunction;
+};
+
+const CitizenshipModal = ({ onClickNext }: TCitizenshipModal) => {
+    const { data: residenceList, isLoading: residenceListLoading } = useResidenceList();
+    const { data: clientCountry, isLoading: clientCountryLoading } = useClientCountry();
+    const [isCheckBoxChecked, setIsCheckBoxChecked] = useState(false);
+    const { values, setFieldValue } = useFormikContext<TFirstSignupFormValues>();
+    const isBrazil = values.country === 'br';
```

Comment:
This is a business logic, shouldnt be here. Lets move it to a helper function and name it like isCVMEnabled or something

Reply:
Can check eu_countries in utility